portafly/src/i18n/setupTranslate.tsx

```diff
@@ -0,0 +1,8 @@
+import Polyglot from 'node-polyglot'
+
+export const setupTranslate = (polyglot:Polyglot , locale: string, phrases: object) => {
```

Comment:
Maybe we should define a type for this? I was thinking something like
```
type Phrases = {
  [key: string]: string
}
```
```suggestion
export const setupTranslate = (polyglot:Polyglot , locale: string, phrases: Phrases) => {
```

Reply:
In that case it should be 
```
type Phrases = {
  [key: string]: string | object
}
```